frb_codegen/src/generator/dart/mod.rs

```diff
@@ -453,12 +453,20 @@ fn generate_api_func(func: &IrFunc, ir_file: &IrFile) -> GeneratedApiFunc {
             execute_func_name,
             func.wire_func_name(),
             wire_param_list.join(", "),
-            if func.name.contains(STATIC_METHOD_MARKER) {
+            if func.name.contains(STATIC_METHOD_MARKER)
+                && func.name.split("___").last().unwrap() == {
```

Comment:
nit: `___` sounds like some magic constant

Shall we encapsulate all logic about function naming into one place? For example, this place will encode/decode a string (the final function name) from/to a struct (say, `struct DecodedFunctionName {real_name: String, is_static_method: bool, struct_name: str, ...}`)

Reply:
Ok, I think I can create like `is_static_method` and `split_static_method`

Reply:
Sure. The main idea is, a function name now indeed contains many different pieces of information, such as "real function name", "is it static", "struct name", etc. So it is indeed a classical serialization thing.